AlpheusCore/Logger.fs

```diff
@@ -0,0 +1,26 @@
+﻿module ItisLab.Alpheus.Logger
+
+let internal showVerbose = true
+
+type LogCategory =
+    | Test 
+    | Debug
```

Comment:
Let's remove Debug as it more close to verbosity level rather than category?

Reply:
Agree.